backend/src/activitypub/objects/index.ts

```diff
@@ -231,7 +231,7 @@ export async function sanitizeObjectProperties(properties: unknown): Promise<APO
  * See https://docs.joinmastodon.org/spec/activitypub/#sanitization
  */
 export async function sanitizeContent(unsafeContent: string): Promise<string> {
-	return await contentRewriter.transform(new Response(unsafeContent)).text()
+	return await getContentRewriter().transform(new Response(unsafeContent)).text()
```

Comment:
Out of curiosity, why now create a new rewriter every time?

Reply:
yeah we shouldn't need.

Reply:
ah yeah, good spot! 😄 .... I was thinking that this might need a comment 😅 (but I didn't just to keep the code cleaner 😓)

The problem is that we import backend stuff from the frontend and this file is getting imported as well (for the account page). I was getting the error that HTMLRewriter is not defined, since it is not in the browser, so I wrapped the creation in these functions so that they HTMLRewriter gets created only when needed (so only in the backend) solving the issue

___

Actually, we shouldn't import stuff from the backend right?

We do it in a handful of places:
![Screenshot 2023-02-03 at 13 44 49](https://user-images.githubusercontent.com/61631103/216618980-18d90a96-ab59-4d6c-a81d-69b1d0e9ae26.png)

I think it would be great if we could avoid that, maybe we can create a shared library which backend and frontend can import both from? 🤔